### PR TITLE
Motoki/individual page: bare minimum layout

### PR DESCRIPTION
**Pull request by motokikasai**
I've used 'search' screen as the sandbox for my task but reversed it so you will NOT see the individualPageScreen linked anywhere as of now.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: screens/SearchScreen.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import ShareIcon from '../components/share-app-component;
+import ShareIcon from '../components/share-app-component';
```
Comment (Julianb18) [inline screens/SearchScreen.js:3]: What is the single quotation mark at the end is it supposed to be there?

**Review by Julianb18 — CHANGES_REQUESTED** [review]
Just one thing can we always stick to naming our components/arrow functions with caps so IndividualPage and export under the styling. Just so we have similar code across all pages.

**Comment by motokikasai** [comment]
Function name + export name have been capitalized.

**Review by Julianb18 — APPROVED** [review]
Looks awesome!

**Pull request merged**